### PR TITLE
Added Polkadot cross-chain transfer guide

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

File: src/substrate-bridge.md [hunk 1]
```diff
@@ -193,6 +193,59 @@ Only specific XCM messages for/from the corresponding networks have been tested
 
 :::
 
+#### SORA <-> Polkadot
+
+This section showcases various scenarios for bidirectional transfers between SORA mainnet and Polkadot.
```
Comment (WRRicht3r) [inline src/substrate-bridge.md]:
```suggestion
This section showcases various scenarios for bidirectional transfers between the SORA mainnet and Polkadot.
```

File: src/substrate-bridge.md [hunk 2]
```diff
@@ -193,6 +193,59 @@ Only specific XCM messages for/from the corresponding networks have been tested
...
+
+##### Polkadot -> SORA mainnet
+
+To transfer assets from Polkadot to the SORA network, additional to the `XOR` for the SORA network transaction, you will need to hold `DOT` tokens to pay for the Kusama network transaction fees.
```
Comment (WRRicht3r) [inline src/substrate-bridge.md]:
```suggestion
To transfer assets from Polkadot to the SORA network, in addition to the `XOR` for the SORA network transaction, you will need to hold `DOT` tokens to pay the Polkadot network transaction fees.
```

File: src/substrate-bridge.md [hunk 3]
```diff
@@ -193,6 +193,59 @@ Only specific XCM messages for/from the corresponding networks have been tested
...
+
+To fill the call data with information from the figure, follow this [link](https://polkadot.js.org/apps/?rpc=wss://rpc.polkadot.io#/extrinsics/decode/0x95028400ccba123cc29c8e6464bb6d5d51236a1d9b4c5b61b86061850a7e85434833203501f683e251cc059cf488abe703192669ba3f6d2440e8fc9740cf85d8f0924a245916133f564d1fe0b248c97ef51008ce52fe1442856b6699283d20b28c5ed3a78f5501350100630203000100a51f0300010100ccba123cc29c8e6464bb6d5d51236a1d9b4c5b61b86061850a7e85434833203503040000000007000c77420300000000). Make sure you modify the recipient as well as the amount of tokens to be sent, if needed.
+
+![Image: Example of a DOT transfer to the Polkadot network from the SORA mainnet](./assets/bridgeTransferPolkadotToSora.png)
+
+The extrinsic to submit is `xcmPallet` then `reserveTransferAssets(dest,beneficiary,assets,feeAssetitem)`:
+
+- The dest: is `V3`
+  - The `interior:XCMV3Junctions` is `X1`
+  - The `Parachain` is `2035`
```
Comment (WRRicht3r) [inline src/substrate-bridge.md]:
```suggestion
  - The `Parachain` is `2025`
```

Comment (WRRicht3r) [reply src/substrate-bridge.md]: Isn't the SORA Polkadot parachain ID 2025?

File: src/substrate-bridge.md [hunk 4]
```diff
@@ -193,6 +193,59 @@ Only specific XCM messages for/from the corresponding networks have been tested
...
+- Then in the next section, `Beneficiary:XCMVersionedMultiLocation` is `V3`
+  - Under `V3:XCMMultilocation` the `interior:XCMV3Junctions` is `X1`
+  - The `X1:XCMV3Junction` is `Accountid32` and in the `id:[u8:32]` section, input your SORA network account address (as the recipient of the to transfer)
```
Comment (WRRicht3r) [inline src/substrate-bridge.md]:
```suggestion
  - The `X1:XCMV3Junction` is `Accountid32`, and in the `id:[u8:32]` section, input your SORA network account address (as the recipient of the transfer)
```

File: src/substrate-bridge.md [hunk 5]
```diff
@@ -193,6 +193,59 @@ Only specific XCM messages for/from the corresponding networks have been tested
...
+- So far, you now have the sender and the receiver for the transfer, now let’s input the asset.
+  - In `assets`: `XcmVersionedMultiAssets` select `V3`, then click `Add Item`
+  - In the field `Fungible:Compact<128>` add the number of tokens to be transferred. You have to add ten zeroes after the amount.
```
Comment (WRRicht3r) [inline src/substrate-bridge.md]:
```suggestion
  - In the field `Fungible:Compact<128>` add the number of tokens to be transferred. **You have to add ten zeroes after the amount**.
```

File: src/substrate-bridge.md [hunk 6]
```diff
@@ -193,6 +193,59 @@ Only specific XCM messages for/from the corresponding networks have been tested
...
+
+Finally, submit the transaction and sign on the pop-up using your password. After the transaction is successful, you will notice your `DOT` balance is reduced in Kusama and has increased proportionally in your SORA account. Now let’s test a transaction in the opposite direction;
```
Comment (WRRicht3r) [inline src/substrate-bridge.md]:
```suggestion
Finally, submit the transaction and sign on the pop-up using your password. After the successful transaction, you will notice your `DOT` balance is reduced in Polkadot and has increased proportionally in your SORA account. Now, let’s test a transaction in the opposite direction;
```

File: src/substrate-bridge.md [hunk 7]
```diff
@@ -193,6 +193,59 @@ Only specific XCM messages for/from the corresponding networks have been tested
...
+
+- The `networkId` is `Sub` since we are doing a transfer to another Substrate-based chain
+  - As a `Sub` type choose `Kusama`
+  - As an `assetId`, enter the `DOT` token address `0x0003b1dbee890acfb1b3bc12d1bb3b4295f52755423f84d1751b2545cebf000b`
+- Moving on to the recipient configuration;
+  - Choose `Parachain`
+  - Select version is `V3`
+  - Parent is `1` to target Kusama
+- Enter your Kusama network address as `accountId`
+- Finally, enter the desired `amount` to be sent. You have to add eighteen zeroes after the amount.
```
Comment (WRRicht3r) [inline src/substrate-bridge.md]:
```suggestion
- Finally, enter the desired `amount` to be sent. **You have to add eighteen zeroes after the amount**.
```

File: src/substrate-bridge.md [hunk 8]
```diff
@@ -193,6 +193,59 @@ Only specific XCM messages for/from the corresponding networks have been tested
...
+
+Then, submit the transaction and sign on the pop-up using your password. After the transaction is successful, you will notice your `KSM` balance is reduced in the SORA network and increased in Kusama.
```
Comment (WRRicht3r) [inline src/substrate-bridge.md]:
```suggestion
Then, submit the transaction and sign on the pop-up using your password. After the successful transaction, you will notice your `DOT` balance is reduced in the SORA network and increased in Polkadot.
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

File: src/substrate-bridge.md [hunk 9]
```diff
@@ -193,6 +193,59 @@ Only specific XCM messages for/from the corresponding networks have been tested
...
+![Image: Example of a DOT transfer to the SORA mainnet from the Polkadot relay chain](./assets/bridgeTransferSoraToPolkadot.png)
+
+The extrinsic to submit is `bridgeProxy` then `burn`:
+
+- The `networkId` is `Sub` since we are doing a transfer to another Substrate-based chain
+  - As a `Sub` type choose `Kusama`
+  - As an `assetId`, enter the `DOT` token address `0x0003b1dbee890acfb1b3bc12d1bb3b4295f52755423f84d1751b2545cebf000b`
+- Moving on to the recipient configuration;
+  - Choose `Parachain`
+  - Select version is `V3`
```
Comment (outoftardis) [inline src/substrate-bridge.md]:
```suggestion
  - The version is `V3`
```


```suggestion
  - For the version select `V3`
```

**Review by WRRicht3r — APPROVED** [review]

**Review by outoftardis — APPROVED** [review]

**Pull request merged**